Code/Source/solver/mat_models.cpp

```diff
@@ -188,116 +360,120 @@ void get_pk2cc(const ComMod& com_mod, const CepMod& cep_mod, const dmnType& lDmn
   }
 
   // Electromechanics coupling - active strain
-  auto Fe  = F;
-  auto Fa = mat_id(nsd);
-  auto Fai = Fa;
-
-  if (cep_mod.cem.aStrain) {
-    actv_strain(com_mod, cep_mod, ya, nfd, fl, Fa);
-    Fai = mat_inv(Fa, nsd);
-    Fe = mat_mul(F, Fai);
-  }
-
-  double J = mat_det(Fe, nsd);
+  EigenMatrix<nsd> Fe  = F;
+  EigenMatrix<nsd> Fa = EigenMatrix<nsd>::Identity();
+  EigenMatrix<nsd> Fai = Fa;
+
+  // if (cep_mod.cem.aStrain) {
+  //   actv_strain(com_mod, cep_mod, ya, nfd, fl, Fa);
+  //   Fai = Fa.inverse();
+  //   Fe = F * Fai;
+  // }
```

Comment:
Remove commented-out code.

Reply:
@aabrown100-git If you are going to leave in commented-out code then be sure to document it so developers can understand why it is there.

Reply:
Got it, will leave a detailed comment